easybuild/easyblocks/generic/binary.py

```diff
@@ -131,9 +134,10 @@ def sanity_check_rpath(self):
                       self.__class__.__name__)
 
     def make_module_extra(self):
-        """Add the install directory to the PATH."""
+        """Add the specified directory to the PATH."""
 
         txt = super(Binary, self).make_module_extra()
-        txt += self.module_generator.prepend_paths("PATH", [''])
+        if not self.cfg['prepend_to_path'] is None:
```

Comment:
@Flamefire Please use `if ... is not None`, see https://docs.python.org/2/reference/expressions.html#not-in